src/test/mutation/highlights-update.integration.ts

```diff
@@ -0,0 +1,116 @@
+import { ApolloServer } from 'apollo-server-express';
+import { getServer } from '../../server';
+import sinon from 'sinon';
+import { ContextManager } from '../../context';
+import { readClient } from '../../database/client';
+import { seedData } from '../query/highlights-fixtures';
+import { UPDATE_HIGHLIGHT } from './highlights-mutations';
+import { HighlightEntity, HighlightInput } from '../../types';
+
+describe('Highlights creation', () => {
```

Comment:
Should this be 'Highlights update' instead of 'creation'?

Reply:
catching all my copy-paste errors :)